roles/setup_sushy_tools/templates/sushy-emulator.conf.j2

```diff
@@ -43,7 +43,7 @@ SUSHY_EMULATOR_LIBVIRT_URI = u'qemu:///system'
 # Instruct the libvirt driver to ignore any instructions to
 # set the boot device. Allowing the UEFI firmware to instead
 # rely on the EFI Boot Manager
-SUSHY_EMULATOR_IGNORE_BOOT_DEVICE = True
+SUSHY_EMULATOR_IGNORE_BOOT_DEVICE = {{ sushy_ignore_boot_device | bool | ternary("True", "False") }}
```

Comment:
I would suggest doing this when dealing with the ternary I've had some weird behaviour with ternary because of the order things get invoked. 
```suggestion
SUSHY_EMULATOR_IGNORE_BOOT_DEVICE = {{ (sushy_ignore_boot_device | bool) | ternary("True", "False") }}
```